### ISSUE TITLE
Name.deb is not a package.

### ISSUE THREAD

**Issue by CallieG** [donadigo/eddy#26]
So, I am having some weird issues with Eddy, in that no matter what I do, I cannot use it to install any debs whatsoever.

![eddy](https://cloud.githubusercontent.com/assets/21113574/26756335/e1b9bf9c-4854-11e7-938e-cbbaf1f71fbe.png)
![eddy2](https://cloud.githubusercontent.com/assets/21113574/26756336/e39b7260-4854-11e7-9328-593cb5fd5bf6.png)

**Comment by donadigo** [comment]
Can you tell me on what system are you running? It seems like Eddy couldn't determine the extension of preferred package format.

**Comment by CallieG** [comment]
Elementary Loki, 0.4.1

**Comment by donadigo** [comment]
Hm, can you copy-paste me what is the output of `com.github.donadigo.eddy --debug` when you run it in your terminal? Run it, open the file, and when the dialog comes up, paste the output here. It would be greatly appreciated.

**Comment by CallieG** [comment]
I will try again later.  Having some difficulties not tech related.

**Comment by muhajirframe** [comment]
same here problem here, trying to install atom text editor.

`eddy --debug` gives me this 

```
eddy --debug
[INFO 21:12:27.074247] Application.vala:153: Eddy version: 1.1.4
[INFO 21:12:27.074314] Application.vala:155: Kernel version: 4.4.0-79-generic
[WARNING 21:12:27.155059] [Gtk] Theme parsing error: gtk-widgets.css:2045:14: 'min-height' is not a valid property name
[WARNING 21:12:27.155112] [Gtk] Theme parsing error: gtk-widgets.css:2046:13: 'min-width' is not a valid property name
[WARNING 21:12:27.155166] [Gtk] Theme parsing error: gtk-widgets.css:2051:13: 'min-width' is not a valid property name
[DEBUG 21:12:27.176252] [Gtk] Connecting to session manager
[DEBUG 21:12:27.247922] [PackageKit] notify::connected
[INFO 21:12:34.907004] [Gtk] GtkDialog mapped without a transient parent. This is discouraged.
```

**Comment by CallieG** [comment]
Hey!  Sorry it took so long, but here it is:

eddy --debug
[INFO 17:10:34.185090] Application.vala:153: Eddy version: 1.1.4
[INFO 17:10:34.185353] Application.vala:155: Kernel version: 4.4.0-78-generic
[WARNING 17:10:34.238597] [Gtk] Theme parsing error: gtk-widgets.css:2045:14: 'min-height' is not a valid property name
[WARNING 17:10:34.238697] [Gtk] Theme parsing error: gtk-widgets.css:2046:13: 'min-width' is not a valid property name
[WARNING 17:10:34.238800] [Gtk] Theme parsing error: gtk-widgets.css:2051:13: 'min-width' is not a valid property name
[DEBUG 17:10:34.281464] [Gtk] Connecting to session manager
[DEBUG 17:10:34.427248] [PackageKit] notify::connected

[INFO 17:12:13.429908] [Gtk] GtkDialog mapped without a transient parent. This is discouraged.

**Comment by CallieG** [comment]
Oops.

eddy --debug
[INFO 17:10:34.185090] Application.vala:153: Eddy version: 1.1.4
[INFO 17:10:34.185353] Application.vala:155: Kernel version: 4.4.0-78-generic
[WARNING 17:10:34.238597] [Gtk] Theme parsing error: gtk-widgets.css:2045:14: 'min-height' is not a valid property name
[WARNING 17:10:34.238697] [Gtk] Theme parsing error: gtk-widgets.css:2046:13: 'min-width' is not a valid property name
[WARNING 17:10:34.238800] [Gtk] Theme parsing error: gtk-widgets.css:2051:13: 'min-width' is not a valid property name
[DEBUG 17:10:34.281464] [Gtk] Connecting to session manager
[DEBUG 17:10:34.427248] [PackageKit] notify::connected

[INFO 17:12:13.429908] [Gtk] GtkDialog mapped without a transient parent. This is discouraged.

**Comment by donadigo** [comment]
This should be fixed since 1.1.5. If the problem still exists, please reopen.
